Integrations/integration-RsaNetwitnessSecurityAnalytics.yml

```diff
@@ -1081,57 +1297,60 @@ script:
-        case 'nw-get-components':
+        case 'netwitness-im-get-components':
             var components = getComponents(sessionId, args);
-            results = { components: components };
+            results = buildRetValGetComponents(components);
             break;
-        case 'nw-get-events':
+        case 'netwitness-im-get-events':
             var events = getEvents(sessionId, args);
-            results = { events: events };
+            results = buildRetValGetEvents(events);
             break;
-        case 'nw-get-event-details':
-            results = getEventDetails(sessionId, args);
+        case 'netwitness-im-get-event-details':
+            event = getEventDetails(sessionId, args);
+            results = buildRetValGetEventDetails(event);
             break;
-        case 'nw-get-alerts':
+        case 'netwitness-im-get-alerts':
             var alerts = filterAlerts(sessionId, args);
-            results = { alerts: alerts };
+            results = buildRetValGetAlerts(alerts);
             break;
-        case 'nw-get-alert-details':
+        case 'netwitness-im-get-alert-details':
             var alert = getAlertById(sessionId, args);
-            results = { alert: alert };
+            results = buildRetValGetAlert(alert);
             break;
-        case 'nw-get-alert-original':
+        case 'netwitness-im-get-alert-original':
             var alert = getOriginalAlertById(sessionId, args);
-            results = { alert: alert };
+            results = buildRetValGetAlertOrig(alert);
             break;
-        case 'nw-get-available-assignees':
+        case 'netwitness-im-get-available-assignees':
             var availableAssignees = getAvailableAssignees(sessionId);
-            results = { availableAssignees: availableAssignees };
+            results = buildRetValAvailableAssignees(availableAssignees);
             break;
-        case 'nw-create-incident':
+        case 'netwitness-im-create-incident':
             var availableAssignees = getAvailableAssignees(sessionId);
             var newIncident = createIncident(sessionId, args, incidentManagementId, availableAssignees);
-            results = { incident: newIncident };
+            results = buildRetValCreateIncident(newIncident);
             break;
-        case 'nw-add-events-to-incident':
+        case 'netwitness-im-add-events-to-incident':
+            var md = [];
             var availableAssignees = getAvailableAssignees(sessionId, args);
             var isSuccess = addEventsToIncident(sessionId, args, incidentManagementId, availableAssignees);
-            results = { success: isSuccess };
+            md.push({ success: isSuccess })
+            results = {Type: entryTypes.note, Contents: { success: isSuccess }, ContentsFormat: formats.json, HumanReadable: tableToMarkdown(command, md), ReadableContentsFormat: formats.markdown};
             break;
-        case 'nw-update-incident':
+        case 'netwitness-im-update-incident':
```

Comment:
![image](https://user-images.githubusercontent.com/7270217/28249348-e064f90c-6a5c-11e7-889a-ffe680352b37.png)

You should follow the design https://github.com/demisto/server/issues/7102
![image](https://user-images.githubusercontent.com/7270217/28249352-ef4525e6-6a5c-11e7-8df7-0b1fe345955b.png)